### PR TITLE
#19 - Added package content location to registration leaves

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #19 
Added `packageContent` URL to registration leaf

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#66](https://github.com/artipie/nuget-adapter/issues/66) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/3eeab706-0669-479d-ac1c-fdbbbdc14570); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artpie/nuget/http/metadata/ContentStorage.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import com.artpie.nuget.PackageIdentity;
+import java.net.URL;
+
+/**
+ * Package content storage.
+ *
+ * @since 0.1
+ */
+public interface ContentStorage {
```
Comment (Vatavuk) [inline src/main/java/com/artpie/nuget/http/metadata/ContentStorage.java:34]: @olegmoz the class name is misleading. It looks like an implementation of `Storage` interface. How about `ContentLocation`?

Comment (olegmoz) [reply src/main/java/com/artpie/nuget/http/metadata/ContentStorage.java]: @Vatavuk agree, this name is better. Renamed it, please review

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will at least 3 issues during next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**